docs/transactions.md

```diff
@@ -121,16 +121,20 @@ A summary of KV store and paths used:
     - Certificate uniqueness:
         - `6:<Certificate's Subject>:<Certificate's Subject Key ID>` : bool
 - KV store name: `model`
-    - Model Infos 
+    - Model  
         - `1:<vid>:<pid>` : `<model info>`
+    - Model  Versions
+        - `2:<vid>:<pid>` : ` list of model versions`
```

Comment:
Are there any reasons to use the same prefix `2` for `list of model versions` and `Model  Version`?
I think using different prefixes could be better, as in future we may implement queries like `give all values for keys started with 2:<vid>:<pid>`. Currently it will return not only all model versions, but also an entity with a lits of all model versions.